templates/data/mongodb/managed.html

```diff
@@ -0,0 +1,466 @@
+            <p>
+              Our team manages all your 'day 2' operations, from observability to upgrades. We handle all issues, and consistently re-enforce the security of your environment.
+            </p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">3. Optional handover</h3>
+            <div class="p-image-wrapper u-hide--small" style="padding:10%;">
+              {{ image(url="https://assets.ubuntu.com/v1/3a4d4155-Training.svg",
+                            alt="",
+                            width="300",
+                            height="150",
+                            hi_def=True,
+                            attrs={"class": "p-image-container__image"},
+                            loading="lazy") | safe
+              }}
+            </div>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              There's no vendor lock-in with Canonical, so if after a while you want to start managing your environments by yourself, we can teach you how and give you remote support for high-severity cases.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Managed NoSQL databases for big data and AI-powered applications</h2>
+      </div>
+      <div class="col">
+        <a href="https://ubuntu.com/engage/managed-nosql">
+          <div class="p-image-wrapper">
+            <div class="p-image-container u-hide--small">
+              {{ image(url="https://assets.ubuntu.com/v1/b252a9b1-generic-webinar-thumbnail.png",
+                            alt="",
+                            width="1800",
+                            height="1015",
+                            hi_def=True,
+                            attrs={"class": "p-image-container__image"},
+                            loading="lazy") | safe
+              }}
+            </div>
+          </div>
+        </a>
+        <p>
+          Watch the webinar for a deep dive into the advantages of adopting a managed NoSQL database for key big data and AI use cases.
+        </p>
+        <p>Learn what operational excellence means for NoSQL databases, and how you can achieve it.</p>
+        <div class="p-section--shallow">
+          <hr class="is-muted" />
+          <p>
+            <a href="https://ubuntu.com/engage/managed-nosql">Watch the webinar to learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Why choose Managed MongoDB?</h2>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided u-no-margin--bottom">
+          <li class="p-list__item is-ticked">
```

Comment:
<img width="557" alt="Screenshot 2024-08-22 at 1 44 34 PM" src="https://github.com/user-attachments/assets/52ea8fb0-7e1a-4d9a-806b-43984c1d51a3">
Can you please align the tick icon to the paragraph text? There is also extra bottom padding on each paragraph list item, I think wrapping the `p` element in `p-list__item` introduces the additional spacing.